e2e-test/log/src/integrationTest/java/org/opensearch/dataprepper/integration/log/ParallelGrokStringSubstituteLogTest.java

```diff
@@ -69,24 +67,28 @@ public void testPipelineEndToEnd() throws JsonProcessingException {
                     substRequest.source(
                             SearchSourceBuilder.searchSource().size(100)
                     );
+                    // Verify documents from grok index
                     final SearchResponse grokResponse = restHighLevelClient.search(grokRequest, RequestOptions.DEFAULT);
                     final List<Map<String, Object>> grokSources = getSourcesFromSearchHits(grokResponse.getHits());
+
                     Assert.assertEquals(1, grokSources.size());
-		    Map<String, Object> grokSource = grokSources.get(0);
+                    Map<String, Object> grokSource = grokSources.get(0);
                     Assert.assertEquals(4, grokSource.size());
-		    Assert.assertEquals(grokSource.get("message"), testString);
-		    String[] words = testString.split(" ");
-		    Assert.assertEquals(grokSource.get("word1"), words[0]);
-		    Assert.assertEquals(grokSource.get("word2"), words[1]);
-		    Assert.assertEquals(grokSource.get("word3"), words[2]);
+                    Assert.assertEquals(TEST_STRING, grokSource.get("message"));
+                    String[] words = TEST_STRING.split(" ");
+                    Assert.assertEquals(words[0], grokSource.get("word1"));
+                    Assert.assertEquals(words[1], grokSource.get("word2"));
+                    Assert.assertEquals(words[2], grokSource.get("word3"));
 
+                    // Verify documents from substitute index
                     final SearchResponse substResponse = restHighLevelClient.search(substRequest, RequestOptions.DEFAULT);
                     final List<Map<String, Object>> substSources = getSourcesFromSearchHits(substResponse.getHits());
+
                     Assert.assertEquals(1, substSources.size());
-		    Map<String, Object> substSource = substSources.get(0);
+                    Map<String, Object> substSource = substSources.get(0);
                     Assert.assertEquals(1, substSource.size());
-		    String expectedString = testString.replace("word", "WORD");
-		    Assert.assertEquals(substSource.get("message"), expectedString);
+                    String expectedString = TEST_STRING.replace("word", "WORD");
+                    Assert.assertEquals(expectedString, substSource.get("message"));
```

Comment:
This is one of the reasons I prefer to use Hamcrest - you know what is expected and actual based on the fluent methods.

Reply:
I agree, but it's not required for now. We can clean up Junit asserts later.